baywatch.install

```diff
@@ -324,3 +324,33 @@ function baywatch_update_8013() {
   $baywatch = new BaywatchOperation();
   $baywatch->enable_tide_ui_restriction();
 }
+
```

Comment:
@mayurngondhkar This looks good to me but won't handle removing the config files (i.e. [elasticsearch_connector.cluster.elasticsearch_bay.yml](https://github.com/dpc-sdp/content-vic-gov-au/blob/develop/config/dev/elasticsearch_connector.cluster.elasticsearch_bay.yml)) 

Is that addressed somewhere else?

Reply:
Addressed in "sdp-ansible-upgrade"